src/inference/support/hmc_core.jl

```diff
@@ -221,28 +221,30 @@ function _hmc_step(θ::AbstractVector{<:Real},
     τ = max(1, round(Int, λ / ϵ))
     return _hmc_step(θ, lj, lj_func, grad_func, H_func, τ, ϵ, momentum_sampler;
                      rev_func=rev_func, log_func=log_func)
-end
-
 
-# TODO: remove used Turing-wrapper functions
+end
 
 # Ref: https://github.com/stan-dev/stan/blob/develop/src/stan/mcmc/hmc/base_hmc.hpp
 function find_good_eps(model, spl::Sampler{T}, vi::VarInfo) where T
-    logpdf_func_float = gen_lj_func(vi, spl, model)
+    lj_func = gen_lj_func(vi, spl, model)
     momentum_sampler = gen_momentum_sampler(vi, spl)
+    grad_func = gen_grad_func(vi, spl, model)
     H_func = gen_H_func()
+    θ = vi[spl]
+    ϵ = _find_good_eps(θ, lj_func, grad_func, H_func, momentum_sampler)
+    @info "\r[$T] found initial ϵ: $ϵ"
```

Comment:
Maybe move this to line: 283?

Reply:
In fact, I did this initially. But later moved it for two reasons: 1) `T` is not defined for `_find_good_eps`; 2) maybe it's better to keep functions starting with `_` not printing any information.